### PR TITLE
Add `toml` and `pygments` to `requirements.txt`

### PR DESCRIPTION
**Pull request by littlewhitecloud**
image
image
Really a good project, but it seems that the `requirements.txt` lost `toml`.
@rdbende Can you check it out?

### CODE REVIEW

**Comment by littlewhitecloud** [comment]
Maybe we should, for someone who wants to make a contribution.

**Comment by littlewhitecloud** [comment]
That's fine, but I prefer to execute `pip install -r requirements.txt` and edit on the current path.

File: README.md [hunk 1]
```diff
@@ -8,9 +8,20 @@ Chlorophyll provides the `CodeView` widget for tkinter, which is a `Text` widget
 
 
 ## Installation
-
+Install from pip 
 `pip install chlorophyll`
 
+Install from source
+```console
+git clone https://github.com/rdbende/chlorophyll.git
```
Comment (rdbende) [inline README.md:16]:
```suggestion
`pip install chlorophyll`

## Development install

First, fork the repo, then run these commands in your terminal (assuming, you're on Linux or a Mac, I don't know crap about Windows)
```console
git clone https://github.com/your-username/chlorophyll
```

This isn't for installing from source, but for development.

**Comment by littlewhitecloud** [comment]
So we don't need `requirements.txt` now?

**Comment by Moosems** [comment]
I honestly do it the same way as white cloud. Not my repo though :).

**Comment by littlewhitecloud** [comment]
@rdbende Whether to delete or not, it's up to you to decide

**Comment by rdbende** [comment]
> I honestly do it the same way as white cloud. Not my repo though :).

But in a venv?

**Review by rdbende — APPROVED** [review]

**Pull request merged**